src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -193,24 +193,9 @@ <h4 *ngIf="displaySubsection(0, 9)">Setting Up Sessions</h4>
       Not at the moment. But you can mention in the question description that it is a compulsory question (TEAMMATES will not enforce it though).
     </p>
     <tm-example-box *ngIf="questionsToCollapsed[SessionsSectionQuestions.SET_QUESTION_COMPULSORY]" @collapseAnim>
-      <div class="background-color-light-blue">
-        <div class="row padding-15px text-left">
-          <div class="col-md-2">
-            <div class="question-basic-info">Question</div>
-          </div>
-          <div class="col-md-10">
-            <textarea class="form-control" [disabled]="true" placeholder="How many hours did you spend on the project?"></textarea>
-          </div>
-        </div>
-        <div class="row padding-15px text-left">
-          <div class="col-md-2">
-            <div class="question-basic-info">[Optional]<br/>Description</div>
-          </div>
-          <div class="col-md-10">
-            <tm-rich-text-editor [richText]="'<p><strong>This question is </strong><span style=\'color: rgb(255, 0, 0); background-color: rgb(255, 255, 0);\'><strong>compulsory</strong></span><strong>!</strong></p>'" minHeightInPx="100"></tm-rich-text-editor>
-          </div>
-        </div>
-      </div>
+      <tm-question-edit-brief-description-form [briefPlaceholder]="'How many hours did you spend on the project?'" [isBriefDisabled]="true"
```

Comment:
I'm aware that the previous implementation is that `How many hours did you spend on the project?` is the placeholder, but it looks like the actual value rather than placeholder.

Reply:
Okay I can make this change.